src/Airship/Types.hs

```diff
@@ -140,6 +141,10 @@ instance MonadBaseControl b m => MonadBaseControl b (Webmachine m) where
 -- | A convenience synonym that writes the @Monad@ type constraint for you.
 type Handler m a = Monad m => Webmachine m a
 
+-- | A constraint alias for Airship use cases that require @MonadIO@,
+-- | @MonadBaseControl@, and @MonadState@ constraints.
+type IOStateControl b s m = (MonadIO m, MonadBaseControl b m, MonadState s m)
```

Comment:
Do we need to expose the `b` parameter here? Are people really going to use it in situations where `b` ≠ `IO`?